app/endpoints.php

```diff
@@ -0,0 +1,498 @@
+<?php
+
+use Utopia\Database\Adapter;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+use Utopia\Http\Http;
+use Utopia\Http\Response;
+use Utopia\Http\Validator\ArrayList;
+use Utopia\Http\Validator\Assoc;
+use Utopia\Http\Validator\Boolean;
+use Utopia\Http\Validator\FloatValidator;
+use Utopia\Http\Validator\Integer;
+use Utopia\Http\Validator\Text;
+
+Http::get('/mock/error')
+    ->groups(['api', 'mock'])
+    ->action(function () {
+        throw new Exception('Mock error', 500);
+    });
+
+Http::get('/v1/ping')
+    ->groups(['api'])
+    ->inject('adapter')
+    ->inject('response')
+    ->action(function (Adapter $adapter, Response $response) {
+        $output = $adapter->ping();
+
+        $response->json([
+            'output' => $output
+        ]);
+    });
+
+Http::get('/v1/databases/:database')
+    ->groups(['api'])
+    ->param('database', '', new Text(MAX_STRING_SIZE, 0))
+    ->inject('adapter')
+    ->inject('response')
+    ->action(function (string $database, Adapter $adapter, Response $response) {
+        $output = $adapter->exists($database, null);
+
+        $response->json([
+            'output' => $output
```

Comment:
@Meldiron why do we need the output key? Lets just return the object as is

Reply:
Will remove output key. Closing this as its duplicate of https://github.com/utopia-php/database-proxy/pull/1#discussion_r1366272217